homeassistant/components/systemmonitor/sensor.py

```diff
@@ -70,6 +66,13 @@
 SIGNAL_SYSTEMMONITOR_UPDATE = "systemmonitor_update"
 
 
+def get_cpu_icon() -> Literal["mdi:cpu-64-bit", "mdi:cpu-32-bit"]:
```

Comment:
Has been added to test the icons in an easier manner

Reply:
maybe decorate with `@cache` since it never changes

Reply:
```
diff --git a/homeassistant/components/systemmonitor/sensor.py b/homeassistant/components/systemmonitor/sensor.py
index ad770eae762..cdab34f71a9 100644
--- a/homeassistant/components/systemmonitor/sensor.py
+++ b/homeassistant/components/systemmonitor/sensor.py
@@ -10,6 +10,7 @@ import os
 import socket
 import sys
 from typing import Any, Literal
+from functools import cache
 
 import psutil
 import voluptuous as vol
@@ -66,11 +67,10 @@ SENSOR_TYPE_MANDATORY_ARG = 4
 SIGNAL_SYSTEMMONITOR_UPDATE = "systemmonitor_update"
 
 
+@cache
 def get_cpu_icon() -> Literal["mdi:cpu-64-bit", "mdi:cpu-32-bit"]:
     """Return cpu icon."""
-    if sys.maxsize > 2**32:
-        return "mdi:cpu-64-bit"
-    return "mdi:cpu-32-bit"
+    return "mdi:cpu-64-bit" if sys.maxsize > 2**32 else "mdi:cpu-32-bit"
 
 
 @dataclass(frozen=True)
```